rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [ ] rental should add itself to the @person achieving the 'belongs-to' association, something like so:
```
    person.rentals << self
```

Reply:
Done Thanks for suggestion.